etc/korrel8r/rules/netflow.yaml

```diff
@@ -0,0 +1,10 @@
+rules:
```

Comment:
Yep - this is very interesting for korrel8r because it will be able to expose "talks-to" relationships based on network traffic, by following paths like: `Pod1 -src-rule-> netflow-event -dst-rule-> Pod2`
To date it does more "structural" relationships like "logged-by", "created-by", "measurement-of" that are more obvious in static console tools.